docs/using-groups.md

```diff
@@ -148,6 +148,14 @@ Where ExampleGroupsPayload is another generated class, and to retrieve variable
    // VariableOne and VariableTwo are the names of your variables inside the map payload, capitalized first character 
 </code></pre>
 
+#### json payload returns
+
+json payloads provide an additional method to attempt retrieving the payload as the provided class type inputted as the parameter:
```

Comment:
I think it's worth mentioning this method throws if the deserialization fails.

Reply:
I think it returns `null` instead of throwing an exception, but sounds it's worth mentioning it anyway.